e2e-tests/batch-mode-tests.sh

```diff
@@ -0,0 +1,97 @@
+java -cp batch/target/fhir-batch-etl-bundled-0.1.0-SNAPSHOT.jar org.openmrs.analytics.FhirEtl --openmrsServerUrl=http://localhost:8099/openmrs --openmrsUserName=admin --openmrsPassword=Admin123  --searchList=Patient,Encounter,Observation --batchSize=20 --targetParallelism=20 --fileParquetPath=/tmp/analytics-tests/
+if [[ -z $(ls /tmp/analytics-tests/Patient) ]]; then
+  echo "${print_prefix} DATA NOT SYNCED TO PARQUET FILES"
+  exit 0
+fi
+echo "${print_prefix} PARQUET FILES SYNCED IN /tmp/analytics-tests DIRECTORY"​
+current_path=`pwd`
+echo "${print_prefix} COPYING PARQUET TOOLS JAR FILE TO ROOT"
+cp e2e-tests/parquet-tools-1.11.1.jar /tmp/analytics-tests/
+if ! [[ $(command -v jq) ]]; then
+    echo "INSTALLING JSON COMMAND-LINE PROCESSOR, JQ" >&2
+    if [[ "$OSTYPE" == "linux-gnu"* ]]; then
+        echo "OSTYPE: linux"
+        sudo apt install jq
+    elif [[ "$OSTYPE" == "darwin"* ]]; then
+        echo "OSTYPE: Mac OSX"
+        brew install jq
+    fi
+fi
+
+if ! [[ $(command -v jq) ]]; then
+  echo "COULD NOT INSTALL JQ, INSTALL MANUALLY TO CONTINUE RUNNING THE TEST" >&2
+  brew install jq
+fi
+cd /tmp/analytics-tests
+mkdir omrs
+echo "${print_prefix} Finding number of patients, encounters and obs in openmrs server"
+curl -L -X GET -u admin:Admin123 http://localhost:8099/openmrs/ws/fhir2/R4/Patient/ >> ./omrs/patients.json
+total_test_patients=$(jq '.total' ./omrs/patients.json)
+echo "${print_prefix} Total openmrs test patients ---> ${total_test_patients}"
+curl -L -X GET -u admin:Admin123 http://localhost:8099/openmrs/ws/fhir2/R4/Encounter/ >> ./omrs/encounters.json
+total_test_encounters=$(jq '.total' ./omrs/encounters.json)
+echo "${print_prefix} Total openmrs test encounters ---> ${total_test_encounters}"
+curl -L -X GET -u admin:Admin123 http://localhost:8099/openmrs/ws/fhir2/R4/Observation/ >> ./omrs/obs.json
+total_test_obs=$(jq '.total' ./omrs/obs.json)
+echo "${print_prefix} Total openmrs test obs ---> ${total_test_obs}"
+echo "${print_prefix} Counting number of patients, encounters and obs sinked to parquet files"
+total_patients_streamed=$(java -jar ./parquet-tools-1.11.1.jar rowcount ./Patient/ | awk '{print $3}')
+echo "${print_prefix} Total patients synched to parquet ---> ${total_patients_streamed}"
+total_encounters_streamed=$(java -jar ./parquet-tools-1.11.1.jar rowcount ./Encounter/ | awk '{print $3}')
+echo "${print_prefix} Total encounters synched to parquet ---> ${total_encounters_streamed}"
+total_obs_streamed=$(java -jar ./parquet-tools-1.11.1.jar rowcount ./Observation/ | awk '{print $3}')
+echo "${print_prefix} Total obs synched to parquet ---> ${total_obs_streamed}"
+if [[ (${total_patients_streamed} == ${total_test_patients}) && (${total_encounters_streamed} == ${total_test_encounters}) && ($total_obs_streamed == ${total_test_obs})]] ;
+then
+  echo "${print_prefix} BATCH MODE WITH PARQUET SYNC EXECUTED SUCCESSFULLY"
+else
+  echo "${print_prefix} BATCH MODE WITH PARQUET SYNC TEST FAILED"
```

Comment:
At this point, we should have `exit 1` or similar so that on CI tools, we fail the build if the batch mode parquet sync test failed.